app/src/main/java/com/example/project_1/ui/viewUser/ViewUserDetailFragment.kt

```diff
@@ -32,6 +33,7 @@ class ViewUserDetailFragment : Fragment() {
         val view = inflater.inflate(R.layout.fragment_view_user_detail, container, false)
         noUserTv = view.findViewById(R.id.txtNoUserFound)
         val addButtonInFragment1:Button = view.findViewById(R.id.btnAdd)
+        val showPostButton:Button = view.findViewById(R.id.btnShowPost)
         val rv:RecyclerView = view.findViewById(R.id.recyclerView)
```

Comment:
use apply block on recycler view to set the manager and adaper as don't need to rv reference again and again.